azkaban-common/src/main/java/azkaban/jobtype/JobTypePluginSet.java

```diff
@@ -171,4 +188,88 @@ public void addPluginJobPropsProcessor(final String jobTypeName,
       JobPropsProcessor jobPropsProcessor) {
     this.pluginJobPropsProcessor.put(jobTypeName, jobPropsProcessor);
   }
+
+  /**
+   * @return The list of users which are not allowed as defaultProxyUsers.
+   */
+  public List<String> getDefaultProxyUsersFilter() {
+    return this.defaultProxyUsersFilter;
```

Comment:
Can we be consistent with our `return` use?

Reply:
Sorry I didn't get it. Can you explain?

Reply:
I might be mistaken, but it looks like we have some methods which are explicitly returning like this one, and some which are not like addDefaultProxyUsersFilter. I didn't even know you didn't have to explicitly return! I thought that was only a groovy thing, not a Java thing.

Reply:
Those which are return type void like "addDefaultProxyUsersFilter", need not have explicit return to be mentioned. And unlike scala where the last line is automatically returned, java needs an explicit return if the return type is not void.

Reply:
Sorry, missed that it was void, thanks.